modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/assignments/Assignments.js

```diff
@@ -45,6 +45,20 @@ const Assignments = (props) => {
 		if (assignmentType === 'user') {
 			setSections(assignments.sectionsData);
 		}
+		else if (assignmentType === 'roleType') {
+			const sectionsData = [];
```

Comment:
I Think that sectionData should be composed in each section component, rather than here, as in User updateSelectedItem method. If we decide otherwise, than we should move the logic in User to here.